changes/2209-masalim2.md

```diff
@@ -0,0 +1 @@
+User-friendly TypeError when a `root_validator` returns `None`.
```

Comment:
Better add a verb at the beginning like
> Raise a user-friendly `TypeError` when a `root_validator` does not return a dict (e.g. `None`).

Otherwise LGTM 👍